/**:
  ros__parameters:
    topic:
      pointcloud_in: "/iv_points" #your input pointcloud topic
      camera_image_in: "/cam" #your camera topic
      pointcloud_out: "/colored_point" #your output pointcloud topic
    camera:
      fx: 1527.58
      fy: 1550.17
      cx: 970.83
      cy: 731.92
    transform:
      R_lc: [0.0, 1.0, 0.0, 
            0.0, 0.0, -1.0, 
            1.0, 0.0, 0.0]  # Use decimal points for doubles
            
      t: [0.0, 0.0, 0.05]